devel-common/src/tests_common/pytest_plugin.py

```diff
@@ -952,7 +952,6 @@ def __exit__(self, type, value, traceback):
             if type is not None:
                 return
 
-            dag.clear(session=self.session)
```

Comment:
This was obstructing the test since dag.clear calls clear_task_instances. Removed it to see how useful it is